CONTRIBUTING.md

```diff
@@ -11,15 +11,16 @@ setup your editor to follow it.
 
 ## Release process
 
-- [Draft new GitHub Release](https://github.com/dalibo/pitrery/releases/new). A
-  version is composed of only two number like `2.3`. There is no patch-release.
 - Checkout latest master.
-- Update CHANGELOG, config.mk, pitrery, archive\_wal and restore\_wal. `make
-  checkversion VERSION=X.Y` should help.
+- Update CHANGELOG, config.mk, pitrery, archive\_wal, restore\_wal and
+  rpm/pitrery.spec. `make checkversion VERSION=X.Y` should help.
 - Commit, tag and push with `make disttag`.
 - Build source tarball and sign it with `make distsign`.
+- Update the created GitHub Release: it was created when the tag was created.
```

Comment:
*when the tag was pushed* me semble plus juste et ça allège la phrase. Qu'en penses-tu ?

Reply:
ça me va, je corrige